src/renderer/components/Cards/ValidatorCard.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import BN from "bn.js";
+
+export interface IValidatorCardProps {
+    title: string;
+    type: string;
+    value: BN | number;
+}
+
+export const ValidatorCard: React.FunctionComponent<IValidatorCardProps> = (
+    props: IValidatorCardProps) => {
+
+    if (props.type==="ROI") {
+        return(
+            <div className="validator-card-container">
+                <h5>{props.title}</h5>
+                <h1 className={props.value>=0 ? "plus":"minus"}>
+                    {props.value>=0 ? "+":"-"}{props.value}%</h1>
+                <h5 className={props.value>=0 ? "plus":"minus"}>{props.type}</h5>
+            </div>
+        );
```

Comment:
I would extract these to separate function.
Like:
- renderROI
- renderBalance
- renderUptime

Because we will have some processing and formatting later on